logic/in_transfer.js

```diff
@@ -31,15 +32,15 @@ let shared;
  * @see Parent: {@link logic}
  * @requires helpers/milestones
  * @requires helpers/slots
- * @param {Database} db
  * @param {ZSchema} schema
+ * @param {Storage} storage
  * @todo Add description for the params
  */
 class InTransfer {
-	constructor(db, schema) {
+	constructor(schema, storage) {
```

Comment:
Can't understand why you change the order of parameters and not just replaced `db` with `storage`.

Reply:
Same as before replaced storage and then removed. Will change.